fury/utils.py

```diff
@@ -1153,6 +1294,38 @@ def array_from_actor(actor, array_name, as_vtk=False):
     return numpy_support.vtk_to_numpy(vtk_array)
 
 
+def normals_to_actor(actor, normals):
+    """Set normals to actor which uses polydata.
+
+    Parameters
+    ----------
+    actor : actor
+    normals : normals, represented as 2D ndarrays (Nx3) (one per vertex)
+
+    Returns
+    -------
+    actor
+
+    """
+    polydata = actor.GetMapper().GetInput()
+    set_polydata_normals(polydata, normals)
+    return actor
+
+
+def tangents_to_actor(actor, tangents):
+    """Set tangents to actor which uses polydata.
+
+    Parameters
+    ----------
+    actor : actor
+    tangents : tangents, represented as 2D ndarrays (Nx3) (one per vertex)
+
+    """
+    polydata = actor.GetMapper().GetInput()
+    set_polydata_tangents(polydata, tangents)
+    return actor
+
```

Comment:
can you replace `actor` with `act` to avoid confusion with the module? See #532

Reply:
Sure, I will.

Reply:
Changed.